modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/ClientConnectionContext.java

```diff
@@ -73,20 +75,22 @@
      */
     public static final ClientListenerProtocolVersion VER_1_7_0 = ClientListenerProtocolVersion.create(1, 7, 0);
 
-    /**
-     * Version 1.7.1. User Attributes introduced. New version is needed because user attributes passed in handshake.
-     */
+    /** Version 1.7.1. User Attributes introduced. New version is needed because user attributes passed in handshake. */
     public static final ClientListenerProtocolVersion VER_1_7_1 = ClientListenerProtocolVersion.create(1, 7, 1);
 
+    /** Version 1.8.0. Timezone fix introduced. New version is needed because timezone passed in handshake. */
+    public static final ClientListenerProtocolVersion VER_1_8_0 = ClientListenerProtocolVersion.create(1, 8, 0);
```

Comment:
I would prefer to avoid adding a new protocol version - future maintenance costs are high, especially because this won't be merged to Apache Ignite.

Can we pass the timezone as a User Attribute? I think it'll better fit the nature of this whole fix.

Reply:
This is a good suggestion. Will do.